### ISSUE TITLE
Implement RZ_CMD_ARG_TYPE_FOLDER

### ISSUE THREAD

**Issue by wargio** [rizinorg/rizin#2727]
Would be nice to have RZ_CMD_ARG_TYPE_FOLDER (including auto completion) which can be very useful.

**Comment by iiiuwioajdks** [comment]
How should it be implemented?

**Comment by wargio** [comment]
essentially instead of type file, it should autocomplete any command expecting directories arg

**Comment by ret2libc** [comment]
@wargio Do you remember any command where this would be useful?
Anyway, @iiiuwioajdks look at `rz_cmd.h` and `cmd_api.c` files, where other `RZ_CMD_ARG_TYPE_*` are defined and used.

**Comment by wargio** [comment]
yes, libyara commands and potentially we could add a command also for folders on the flirt commands.

**Comment by ret2libc** [comment]
Look also at `cautocmpl.c` for implementing the autocompletion.
